apps/library/views/participation.py

```diff
@@ -40,3 +47,17 @@ def perform_create(self, serializer):
                 instance.save()
         except (ValueError, HttpError) as error:
             logger.critical(error, exc_info=True)
+
+        try:
+            send_success = send_application_email(serializer)
+            if send_success:
+                instance.sent_to_email = True
+                instance.save()
+        except (
+            AnymailConfigurationError,
+            AnymailRequestsAPIError,
+            AnymailInvalidAddress,
+            ValueError,
+            AnymailSerializationError,
```

Comment:
Можно указать только эти исключения, используя родителя `AnymailError` других исключений
```suggestion
            AnymailError,
            AnymailConfigurationError,
            ValueError
```